config/kip500-single-node.properties

```diff
@@ -147,3 +147,8 @@ zookeeper.connection.timeout.ms=18000
 # We override this to 0 here as it makes for a better out-of-the-box experience for development and testing.
 # However, in production environments the default value of 3 seconds is more suitable as this will help to avoid unnecessary, and potentially expensive, rebalances during application startup.
 group.initial.rebalance.delay.ms=0
+
+############################# Log Basics #############################
+
+# The directory in which the metadata log is kept. If not set, we default to the first directory given by log.dirs
+metadata.log.dir=/tmp/kafka-metadata-log
```

Comment:
I kind of liked letting this default to the first directory given by `log.dirs`.  If we are going to explicitly set it, I think it should not collide with anything else.

Reply:
Good point. A lot of the configuration was in-flux and this change was made to get the broker and controller to share the same log directory. Essentially it was for a single-node configuration running in two different processes. We need to consolidate the configurations as you noted earlier.

Reply:
As @rondagostino said, I think we can delete this line

Reply:
The config changes are likely superseded by https://github.com/confluentinc/kafka/pull/470

Reply:
Merged #470 and fixed.